server/src/grpc/storage_service/write.rs

```diff
@@ -207,13 +208,14 @@ async fn create_table<Q: QueryExecutor + 'static>(
 
     let instance = &ctx.instance;
 
-    if instance.limiter.should_limit(&plan) {
-        ErrNoCause {
-            code: StatusCode::TOO_MANY_REQUESTS,
-            msg: "Create table limited by reject list",
-        }
-        .fail()?;
-    }
+    instance
+        .limiter
+        .try_limit(&plan)
```

Comment:
There is no `blockRule` of `Create table is blocked`?

Reply:
Actually, there is no such rule to forbid creating tables. However, it may be added in the future.